### PR TITLE
feat: Add new docs.

### LINKED ISSUE #1: [NEEDED] Documentation Writers.
As this is my first advanced API Wrapper i've made. I am in need for developers who are capable of implementing documentation for the `weeby-js` module.

### PR DESCRIPTION
**Pull request by WiseDevHelper**
Fixes #1

### CODE REVIEW

File: .github/workflows/deploy-docs.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Deployment
```
Comment (NTMNathan) [inline .github/workflows/deploy-docs.yml:1]: Looks good, don't forget to fix the npm-publish.yml as that is currently broken :)

File: src/lib/custom.js [hunk 2]
```diff
@@ -7,9 +7,18 @@ const { version } = require("../../package.json");
 */
 class Custom {
     constructor(token, baseURL) {
-        /** @access private */
+        /**
+         * The Weeby-API token
```
Comment (NTMNathan) [inline src/lib/custom.js:11]: Rename `Weeby-API` to `Weeby API` 😛. Applies to the rest of the files that contain `Weeby-API`

**Review by NTMNathan — COMMENTED** [review]
This is a looking good so far! Will merge when the fixes and suggestions are done.

Also, we're using GitHub Pages so Netlify will not necessarily be needed.

File: .github/workflows/deploy-docs.yml (hunk 1 above)
Comment (WiseDevHelper) [reply .github/workflows/deploy-docs.yml:1]: I plan to open a separate PR for that in the future as I believe it is out of scope for this one.

**Comment by NTMNathan** [comment]
That's perfect, i'll merge now :)

**Pull request merged**